source/features/embed-gist-via-iframe.tsx

```diff
@@ -0,0 +1,37 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const embedViaScript = select('.file-navigation-option .select-menu-item')!;
+	const embedViaIframe = embedViaScript.cloneNode(true) as HTMLButtonElement;
+
+	// Modify description to distinguish the two items
+	// This is also consistent with the existing "Clone via HTTPS" and "Clone via SSH" items
+	const viaScript = 'Embed via <script>';
+	select('.file-navigation-option [data-menu-button]')!.textContent = viaScript;
```

Comment:
I saw this now, I don't think we need to update the label here. 

The regular label is fine, `<script` is visible right next to it:

<img width="303" alt="" src="https://user-images.githubusercontent.com/1402241/63282749-b3b22880-c2d9-11e9-847b-ece0f2669e5b.png">

It can be dropped

```suggestion
```

Reply:
If we need to drop this, we should drop the line below as well (i.e. https://github.com/sindresorhus/refined-github/pull/2347/files#diff-202a2bef15a311f406c80dbc2c6e74bfR10-R12 together), because when you click another button and click the "Embed" button again, the label will sync with our modified title and become "Embed via <script>", which can be confusing.

Reply:
The next line fine as is. As a starter the it’s better as simply “embed”